gymManagement/src/main/java/com/angef/gymManagement/dto/ManagementDTO.java

```diff
@@ -0,0 +1,53 @@
+package com.angef.gymManagement.dto;
+
+public class ManagementDTO {
+	    private boolean invoiced;
+	    private boolean access;
+	    
+	    private ManagementDTO() {
+	    }
+
```

Comment:
Este constructor debe tener un argumento de tipo Builder y asignarle a cada campo su valor. ejem. this.invoiced = builder.invoiced;